doc/source/dev/gitwash/dot2_dot3.rst

```diff
@@ -0,0 +1,26 @@
+.. _dot2-dot3:
+
+========================================
+ Two and three dots in difference specs
+========================================
```

Comment:
Is the `=` and title misalignment intentional here?

Reply:
I don't think that document really deserves to exist on its own, but for now I brought it in (from NumPy) anyway. Didn't notice this before, but I'll fix it until this information finds its way into somewhere appropriate.